packages/worker-capacity-manager/package.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "worker-capacity-manager",
+	"version": "1.0.0",
+	"description": "",
```

Comment:
It would be good to include a readme.md on what this package does and the other approaches we tried.
(could be another PR if you want to get this one out)

Reply:
Yep good suggestion